apps/frontend/src/app/PageSettings/DatabaseCard.tsx

```diff
@@ -210,6 +335,38 @@ function DataCard({ index, readOnly }: { index: number; readOnly: boolean }) {
                   {t`DatabaseCard.button.download`}
                 </Button>
               </Grid>
+              <Grid item xs={1}>
```

Comment:
Can you hide these buttons behind our dev-flag please?
https://github.com/search?q=repo%3Afrzyc%2Fgenshin-optimizer%20shouldshowdevcomponents&type=code

Reply:
Just the cloud ones

Reply:
done